packages/vc-export/src/index.ts

```diff
@@ -5,16 +5,7 @@
  * found in the LICENSE file in the root directory of this source tree.
  */
 
-/**
- * @module @kiltprotocol/vc-export
- */
-
-export * as KiltCredentialV1 from './KiltCredentialV1.js'
-export * as KiltAttestationProofV1 from './KiltAttestationProofV1.js'
-export * as KiltRevocationStatusV1 from './KiltRevocationStatusV1.js'
-export * as Presentation from './Presentation.js'
-export * as DidJWT from './DidJwt.js'
-export * as vcjs from './vc-js/index.js'
-export * as Types from './types.js'
-export * as constants from './constants.js'
-export * from './errors.js'
+export * from './suites/index.js'
+export * from './purposes/index.js'
+export * from './documentLoader.js'
+export { validationContexts as contexts } from './context/index.js'
```

Comment:
this package only contains the jsonld-signatures support now. moved this a level up